keps/sig-apps/961-maxunavailable-for-statefulset/README.md

```diff
@@ -375,29 +679,60 @@ No
 ###### Will enabling / using this feature result in non-negligible increase of resource usage (CPU, RAM, disk, IO, ...) in any components?
 The controller-manager will see very negligible and almost un-notoceable increase in cpu usage.
 
+###### Can enabling / using this feature result in resource exhaustion of some node resources (PIDs, sockets, inodes, etc.)?
+
+No.
+
 ### Troubleshooting
 
 ###### How does this feature react if the API server and/or etcd is unavailable?
 The RollingUpdate will fail or will not be able to proceed if etcd or apiserver is unavailable and
 hence this feature will also be not be able to be used.
 
 ###### What are other known failure modes?
-NA
+
+<!--
+For each of them, fill in the following information by copying the below template:
+  - [Failure mode brief description]
+    - Detection: How can it be detected via metrics? Stated another way:
+      how can an operator troubleshoot without logging into a master or worker node?
+    - Mitigations: What can be done to stop the bleeding, especially for already
+      running user workloads?
+    - Diagnostics: What are the useful log messages and their required logging
+      levels that could help debug the issue?
+      Not required until feature graduated to beta.
+    - Testing: Are there any tests for failure mode? If not, describe why.
+-->
+
+In a multi-master setup, when the cluster has skewed CCM, the behaviors may different.
+
+- [Failure mode brief description]
+  - Detection: the `rolling-update-duration` didn't decrease when setting the `maxUnavailable` great than 1 or increased abnormally.
+  - Mitigations: Disable the feature.
+  - Diagnostics: Set the logger level great than 4.
+  - Testing: No testing, because the rolling update duration is hard to measure, it can be impact by a lot of things,
+  like the master performance.
+
+###### What steps should be taken if SLOs are not being met to determine the problem?
 
 ## Implementation History
 
 - KEP Started on 1/1/2019
 - Implementation PR and UT by 8/30
+- Bump to beta at 2023-05-11
```

Comment:
The preferred format is: 
- date in YYYY-MM-DD format: descriptions
see other enhancements.